### ISSUE TITLE
Re-meta-tag training accounts in AWS to backfill for bugs in prod

### ISSUE THREAD

**Issue by ActionAnalytics** [bcgov/cloud-pathfinder#1012]
**Describe the issue**
We are blocked on creating well-formed accounts in AWS at the moment.  We have a pool of lab accounts.  Providing dev accounts for SecOps and Parks will unblock them a bit.

**Which Sprint Goal is this issue related to?**
> Note: 'Milestone' is a ZenHub term that we use synonymously with 'Sprint'. 

The 'Milestone' description (created in ZenHub) should clearly list Sprint Goals. This section should indicate which 'goal' this issue is related to.

**Additional context**
Add any other context, attachments or screenshots

**Definition of done**
- Check in keycloak that these have not been assigned
- Discuss among tech team if this is a viable approach and
- If so, change the tags and provision dev accounts to each team
- Later we can provision them an entire project-set once we have them working again

**Comment by wrnu** [comment]
We found a way to work around the ASEA issues and correctly setup the existing accounts. This issue/work is no longer needed
